src/cleanandgo/EquipmentAndSupplies.java

```diff
@@ -88,12 +88,17 @@ private static void printAnnualCleaningSuppliesExpenses() {
         try{
```

Comment:
This function shoul be moved to the Supplies.java file and be called here as Supplies.printAnnualCleaningSuppliesExpenses()

Reply:
Done.